cli/args/flags.rs

```diff
@@ -3380,8 +3380,8 @@ fn import_map_arg() -> Arg {
 }
 
 fn env_file_arg() -> Arg {
-  Arg::new("env")
-    .long("env")
+  Arg::new("env-file")
+    .long("env-file")
```

Comment:
We can't outright rename it, we should keep the `--env` alias working for now.

Reply:
Right, I i've set it so that --env-file is the .long version now.

Reply:
Okay, so cli can maybe only handle a single .long() version. 

What path forward do you prefer? An entirely new cli entry, named env-file next to the env?